client/src/app/tabs/PropertiesContainer.js

```diff
@@ -131,34 +196,56 @@ class PropertiesContainerWrapped extends PureComponent {
 
     const {
       open,
-      width
+      width,
+      fullWidth
     } = getLayoutFromProps(this.props);
 
     return (
       <div
-        ref={ this.ref }
+        ref={ this.containerRef }
         className={ classNames(
           css.PropertiesContainer,
           className,
           { open }
         ) }
         style={ { width } }>
+
+        <div
+          ref={ this.resizeHandlerRef }
+          className={ classNames(
+            'resize-area',
+            { 'snapped-right': !open },
+            { 'snapped-left': fullWidth },
+          ) }
+          draggable
+          onDragStart={ this.handleResizeStart }
+          onDragEnd={ this.handleResizeEnd }
+        >
+          <div className="resize-handle" />
+        </div>
+
         <div
           className="toggle"
           onClick={ this.handleToggle }
           draggable
           onDragStart={ this.handleResizeStart }
           onDragEnd={ this.handleResizeEnd }
-        >Properties Panel</div>
-        {
-          open &&
-            <div
-              className="resize-handle"
-              draggable
-              onDragStart={ this.handleResizeStart }
-              onDragEnd={ this.handleResizeEnd }
-            ></div>
-        }
+          onMouseOver={ () =>
```

Comment:
Can't we handle these `hover` styles with CSS only?

Reply:
Probably yes, but we are anyway dealing with the selector class changes via the ` adjustHandlerDragStyles` helper.

Reply:
I changed this a bit so the hovering state is handled by CSS only. However, I kept this for the `dragging` styles similarly to how we do for the snapping because I found this easiest.